arches/app/models/migrations/11801_alter_spatialview_geometrynode.py

```diff
@@ -0,0 +1,174 @@
+# Generated by Django 5.2.1 on 2025-05-19 16:11
+
+import django.db.models.deletion
+from django.db import migrations, models
+import pgtrigger
+
+
+drop_legacy_trigger = "drop trigger if exists __arches_trg_update_spatial_views on spatial_views; drop function if exists __arches_trg_fnc_update_spatial_views;"
```

Comment:
Alexei has expressed before preferring to see these indented under the Migration class so that the dependencies are always near the top of the file.